docs/self-managed/zeebe-deployment/configuration/gateway.md

```diff
@@ -340,6 +340,10 @@ To use multi-tenancy, you must set [`authentication.mode`](#zeebegatewayclusters
 as well.
 :::
 
+:::note
+If you are using an embedded gateway, refer to the [broker configuration guide](./broker.md/#zeebebrokergatewaymultitenancy).
+:::
```

Comment:
:+1: This is pretty nice! It makes users aware that this setup is also possible.